google/cloud/bigquery/magics/magics.py

```diff
@@ -593,6 +616,37 @@ def _cell_magic(line, query):
 
     close_transports = functools.partial(_close_transports, client, bqstorage_client)
 
+    if args.send_long_job:
```

Comment:
i'm still unclear why this whole block of the async job handling is here, before other common processing happens. i'd expect all of this logic to be inside `_run_query()`: 
```
if send_long_job:
  start thread and use widget to output
else:
  use blocking while loop
return query_job # either complete or running
```

Reply:
I'm currently investigating how to best update and use `_run_query()` atm!